roles/manage_cnp/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# vars file for manage_cnp
```

Comment:
this file can be removed along with the vars directory for this role

Reply:
removed